test/hotspot/jtreg/serviceability/jvmti/events/NotifyFramePopStressTest/NotifyFramePopStressTest.java

```diff
@@ -23,7 +23,7 @@
 
 /**
  * @test
- * @summary Verifies NotifyFramePop request is cleared if JVMTI_EVENT_FRAME_POP is disabled
+ * @summary JVMTI FRAME_POP event is sometimes missed if NotifyFramePop is called as a method is returning
```

Comment:
Would be good to also add an `@bug` statement.

Reply:
Thanks. Added @bug line.